### PR TITLE
KEYCLOAK-17354 respond to tls CertificateRequest

### PR DESCRIPTION
**Pull request by sjweixel**
Set certificate to respond to CertificateRequest for ldaps and starttls requests when LDAP server requires client certificate verification.

### CODE REVIEW

**Comment by sjweixel** [comment]
Please review.

File: server_admin/topics/user-federation/ldap.adoc [hunk 1]
```diff
@@ -98,6 +98,25 @@ There is a configuration property `Use Truststore SPI` in the LDAP federation pr
 By default, the value is `Only for ldaps`, which is fine for most deployments.  The Truststore SPI will only be used
 if the connection URL to LDAP starts with `ldaps`.
 
+If the LDAP server is setup with Mutual TLS, the server will request a client certificate from Keycloak.
```
Comment (andymunro) [inline server_admin/topics/user-federation/ldap.adoc:101]: Generally use present tense, which sounds more active:  "the server requests"

File: server_admin/topics/user-federation/ldap.adoc [hunk 2]
```diff
@@ -98,6 +98,25 @@ There is a configuration property `Use Truststore SPI` in the LDAP federation pr
...
+
+If using ldaps, specify the client certificate to respond to the client request, start keycloak with following:
```
Comment (andymunro) [inline server_admin/topics/user-federation/ldap.adoc:103]: replace keycloak with {project_name}

File: server_admin/topics/user-federation/ldap.adoc [hunk 3]
```diff
@@ -98,6 +98,25 @@ There is a configuration property `Use Truststore SPI` in the LDAP federation pr
...
+[source]
+----
+
+bin/standalone.sh
+-Dkeycloak.tls.keystore.path=KEYSTORE_PATH
+-Dkeycloak.tls.keystore.password=KEYSTORE_PASSWORD
+----
+If using starttls, specify the client certificate to respond to the client request, start keycloak with the following:
```
Comment (andymunro) [inline server_admin/topics/user-federation/ldap.adoc:111]: replace keycloak with {project_name}

**Comment by andymunro** [comment]
To fix the Waiting for Status problem, use this command:  git pull upstream main

**Pull request closed without merging**